job_watcher/jobwatcher/job_watcher.py

```diff
@@ -5,17 +5,23 @@
 import datetime
 import json
 import os
+import sys
+import time
+from http import HTTPStatus
 from json import JSONDecodeError
 from time import sleep
-from typing import Any
+from typing import Any, Literal, cast
 
-from db.data_models import State
-from db.utils.db_updater import DBUpdater
+import requests
 from kubernetes import client  # type: ignore[import-untyped]
 from kubernetes.client import V1ContainerStatus, V1Job, V1Pod  # type: ignore[import-untyped]
 
 from jobwatcher.utils import logger
 
+StateString = Literal["SUCCESSFUL", "UNSUCCESSFUL", "ERROR", "NOT_STARTED"]
```

Comment:
Should this be defined in the db package?

Reply:
I don't think it's worth importing an entire package for 1 line of code, especially when it is likely to be removed post the investigation into an autogenerated client